egs2/librimix/tse1/local/data.sh

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
```

Comment:
Is it possible to call some stage functions in `librimix/enh` recipe ? (Or just remove `data.sh` in `enh` recipe and just call this new one.) Maybe an unified `data.sh` for similar tasks in one dataset would be better.

Reply:
I prefer maintaining `data.sh` for each sub-recipe, as they have different data requirements. For example, the TSE task requires additional enrollments as input.

> Is it possible to call some stage functions in `librimix/enh` recipe ?

Yes, I simply run through all stages in `librimix/enh1/local/data.sh` before the data preparation for TSE. However, the current `librimix/enh1/local/data.sh` script is not written in a flexible manner, and only supports a fixed number of speakers (num_spk=2) in the mixture. I just fix this part to support num_spk=3 as well.